modules/microshift-install-multus-running-cluster.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-cni-multus.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-multus-installing-on-running-cluster_{context}"]
+= Installing the Multus CNI plugin on an running cluster
+
+If you want to attach additional networks to a pod for high-performance network configurations, you can install the {microshift-short} Multus RPM package. After installation, a host restart is required to recreate all the pods with the Multus annotation.
+
+[IMPORTANT]
+====
+The Multus RPM should not be removed after you install it.
+====
+
+.Procedure
+
+. Install the Multus RPM package by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install microshift-multus
+----
++
+[TIP]
+====
+If you create your custom resources (CRs) before completing your installation of Multus, you can avoid restarting the service a second time to apply them.
```

Comment:
I'm not sure which of these versions best fits the process, but one of them might be a more streamlined version that you could use.
```suggestion
Create your custom resources (CRs) before installing Multus to avoid restarting the service a second time to apply them.
```
```suggestion
Create your custom resources (CRs) before Multus installs completely to avoid restarting the service a second time to apply them.
```
```suggestion
Create your custom resources (CRs) before while installing Multus to avoid restarting the service a second time to apply them.
```

Reply:
Hmm, none of those quite fit because it is literally one commmand to install the RPM. BUT restarting the host is quite disruptive and that's what we're trying to get at here. Went with:

If you create your custom resources (CRs) for additional networks now, you can complete your installation and apply configurations with one restart.